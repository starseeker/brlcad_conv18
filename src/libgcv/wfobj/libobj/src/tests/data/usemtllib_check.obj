# check correct content of usemtllib

v 0 0 0
v 0 1 0
v 1 1 0
v 1 0 0

vt 0 0 0
vt 0 1 0
vt 1 1 0
vt 1 0 0

vn 0 0 0
vn 0 1 0
vn 1 1 0
vn 1 0 0

mtllib foo1 bar1
f 1 2 3
mtllib foo2 bar2
f 1 3 4

v 0 0 1
v 0 1 1
v 1 1 1
v 1 0 1

vt 0 0 1
vt 0 1 1
vt 1 1 1
vt 1 0 1

vn 0 0 1
vn 0 1 1
vn 1 1 1
vn 1 0 1

mtllib ignore1 ignore2
mtllib ignore3 ignore2

mtllib bar1 foo1 #same as face 1
f 5 6 7
mtllib foobar
f 5 7 8
