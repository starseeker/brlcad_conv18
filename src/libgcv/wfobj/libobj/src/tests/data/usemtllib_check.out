vertices {
  <0,0,0,1>
  <0,1,0,1>
  <1,1,0,1>
  <1,0,0,1>
  <0,0,1,1>
  <0,1,1,1>
  <1,1,1,1>
  <1,0,1,1>
}
texture coordinates {
  <0,0,0>
  <0,1,0>
  <1,1,0>
  <1,0,0>
  <0,0,1>
  <0,1,1>
  <1,1,1>
  <1,0,1>
}
normals {
  <0,0,0>
  <0,1,0>
  <1,1,0>
  <1,0,0>
  <0,0,1>
  <0,1,1>
  <1,1,1>
  <1,0,1>
}
face {
  groups: default
  object: ''
  material: ''
  materiallibs: bar1 foo1
  texmap: ''
  texmaplibs: 
  shadow_obj: ''
  trace_obj: ''
  smoothing group: '0'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '0'
  vertices: 1 2 3 
}
face {
  groups: default
  object: ''
  material: ''
  materiallibs: bar2 foo2
  texmap: ''
  texmaplibs: 
  shadow_obj: ''
  trace_obj: ''
  smoothing group: '0'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '0'
  vertices: 1 3 4 
}
face {
  groups: default
  object: ''
  material: ''
  materiallibs: bar1 foo1
  texmap: ''
  texmaplibs: 
  shadow_obj: ''
  trace_obj: ''
  smoothing group: '0'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '0'
  vertices: 5 6 7 
}
face {
  groups: default
  object: ''
  material: ''
  materiallibs: foobar
  texmap: ''
  texmaplibs: 
  shadow_obj: ''
  trace_obj: ''
  smoothing group: '0'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '0'
  vertices: 5 7 8 
}
